docs/code.rst

```diff
@@ -132,6 +132,9 @@ Variable                            Description
 ``{{ latest_date }}``               same as ``{{ ds }}``
 ``{{ ti }}``                        same as ``{{ task_instance }}``
 ``{{ params }}``                    a reference to the user-defined params dictionary
+``{{ var.value }}``                 global defined variables represented as a dictionary
+``{{ var.json }}``                  global defined variables represented as a dictionary
```

Comment:
I'm a little unclear as to how this works based on the docs. If I have a variable with key=foo and value=bar, do I call:

    {{ foo.value }}

Or?